portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,19 +14,61 @@
 
 package com.google.sps.servlets;
 
+import com.google.sps.storage.CommentDatastore;
+import com.google.sps.util.JsonUtil;
+
 import java.io.IOException;
+import java.util.Optional;
+
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that returns some example content. */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private static final String HOME_PAGE_URL = "./index.html";
+
+  private CommentDatastore commentDatastore;
+  private String json;
```

Comment:
since this is only used in doGet, it is more appropriate to make it a local variable

Reply:
Changed it back into a String literal since it is only used once